test/puppetlabs/puppetdb/integration/storeconfigs/deactivated_nodes.clj

```diff
@@ -0,0 +1,33 @@
+(ns puppetlabs.puppetdb.integration.storeconfigs.deactivated-nodes
+  (:require [clojure.test :refer :all]
+            [puppetlabs.puppetdb.integration.fixtures :as int]
+            [me.raynes.fs :as fs]))
+
+(deftest ^:integration deactivated-nodes
+  (with-open [pg (int/setup-postgres)
+              pdb (int/run-puppetdb pg {})
+              ps (int/run-puppet-server [pdb] {})]
+
+    (let [exporter-manifest  "@@notify { 'Hello from exporter': }"
+          test-collection (fn test-collection [should-be-active]
```

Comment:
I kinda like the boolean and laying out the assertions in this case, makes it very easy to follow what's going on